lib/cucumber/cli/rerun_file.rb

```diff
@@ -0,0 +1,29 @@
+module Cucumber
+  module Cli
+    class RerunFile
+      attr_reader :path
+
+      def self.rerun_file?(path)
```

Comment:
if you change this to something more abstract like `#can_read?` then I think we could use this same pattern for the other feature-loading strategies, which would be nice.